session/session.go

```diff
@@ -853,6 +853,10 @@ func createSessionFunc(store kv.Storage) pools.Factory {
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+		err = variable.SetSessionSystemVar(se.sessionVars, variable.MaxAllowedPacket, types.NewStringDatum("67108864"))
```

Comment:
In PR https://github.com/pingcap/tidb/pull/11137, `concat_ws` will need this variable, so I set this variable here for internal sql.